components/resources/brave_components_strings.grd

```diff
@@ -506,9 +506,9 @@
       <message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_DEVICE_LIMIT_REACHED_TEXT" desc="">Your Brave Rewards cannot be verified because you've reached the maximum verified device limit.</message>
       <message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_DEVICE_LIMIT_REACHED_TITLE" desc="">Error: Device limit reached</message>
       <message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_ERROR" desc="">Sorry there was a problem processing your request, please try again.</message>
-      <message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_FLAGGED_WALLET_TEXT_1" desc="">Unfortunately, your Brave Rewards profile has been flagged, and you're not able to verify with a custodian at this time. This flag can sometimes be due to irregular Brave Ads activity, though it can also be a false positive.</message>
-      <message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_FLAGGED_WALLET_TEXT_2" desc="">Note that while your Brave Rewards profile is flagged, you won't be able to verify.</message>
-      <message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_FLAGGED_WALLET_TEXT_3" desc="">No action is required on your part. Your Brave Rewards profile will be automatically reevaluated on a regular basis to determine when (or if) the flag can be removed. Please try again every so often to see if the flag has been removed.</message>
+      <message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_FLAGGED_WALLET_TEXT_1" desc="">Unfortunately, your Brave Rewards profile was flagged because our automated checks found some unexpected Brave Ads activity.</message>
+      <message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_FLAGGED_WALLET_TEXT_2" desc="">While flagged, you may continue to see ads, but won't be eligible to receive payouts. You also won't be able to connect a custodial account to your Brave Rewards profile.</message>
```

Comment:
`While flagged` makes it sound like "While you are flagged" here. Can we say "While the profile is flagged," instead? cc: @Miyayes

Reply:
Updated with new text.